File: bubblesort.java
import java.util.*;
public class BubbleSort {
    public static void Sort(int num[], int length)
    {
        for(int i=0; i<length-1; i++) //Number of turns
        {
        for(int j=0; j<length-1-i; j++)
        {
            if(num[j] > num[j+1]) //Swapping logic
            {
                int tmp = num[j+1]; 
                num[j+1] = num[j];
                num[j] = tmp;
            }
        }
        }
        for(int i=0; i<length; i++) //Print result
        {
            System.out.print(num[i]+", ");
        }
        System.out.println();
    }
    public static void main(String[] args) 
    {
        Scanner in = new Scanner(System.in);
        System.out.println("Enter length of the array:");
        int length=in.nextInt();
        int num[] = new int[length];
        for(int k=0; k<length; k++) //Inputing numbers
        {
            System.out.println("Enter a number");
            num[k] = in.nextInt();
        }
        System.out.println("Sorted Array:");
        Sort(num, length);
    }
}

File: bubblesort.js
const bubbleSort = (a) => {
  let swaps;
  let n = a.length - 1;
  let x = a;

  do {
    swaps = false;
    for (var i = 0; i < n; i++) {
      if (x[i] < x[i + 1]) {
        var temp = x[i];
        x[i] = x[i + 1];
        x[i + 1] = temp;
        swaps = true;
      }
    }
    n--;
  } while (swaps);

  return x;
};

console.log(
  bubbleSort([12, 345, 4, 546, 122, 84, 98, 64, 9, 1, 3223, 455, 23, 234, 213])
);
